osu.Game/Online/Multiplayer/MultiplayerClient.cs

```diff
@@ -70,7 +69,7 @@ protected virtual async Task Connect()
                          {
                              options.Headers.Add("Authorization", $"Bearer {api.AccessToken}");
                          })
-                         .AddNewtonsoftJsonProtocol(options => { options.PayloadSerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore; })
+                         .AddMessagePackProtocol()
```

Comment:
Loose thought - if we're going for debuggability by leaving the server supporting both formats, maybe debug builds should be able to use json too?

Reply:
It would be nice to leave the option in code in some way I guess. Not sure it would be used by anyone apart from me, nor how long it will continue to work as we add more models without testing it directly.

Might be best to just assume it is dropped functionality, until we need it again.